include/ttmlir/Dialect/TTNN/Utils/Utils.h

```diff
@@ -35,6 +35,14 @@ mlir::tt::TensorMemoryLayout toTTTensorMemoryLayout(
 mlir::tt::MemorySpace
 toTTMemorySpace(const mlir::tt::ttnn::BufferType bufferType);
 
+// Get DataType from MemRefType
```

Comment:
You can remove this helper functions. TTNNLayout has getDataType (maybe function name is different) for data type for sure. I'm not sure for layout though but you can add new class method for layout in TTNNLayout also.